quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+- Next steps:
+ - What other features are available to explore in the application?
+
+A Quickstart guide is often confused with a Getting Started  guide. Though both these documents help the users get acquainted with the application they widely differ in their target audience. 
+
+There are instances where a Marketing guide is often confused with a Quickstart and a Getting Started guide. The inline table gives a brief comparison about the differences. 
+
+| |Quickstart Guide|Getting Started Guide| Marketing Guide|
+|--------|----------------|----------------------|---------------|
+ |Target Audience|Domain Experts: Know the problem space and are aware of what they need to do|Beginners:New to problem space and the product|Business Decision Maker: Makes strategic decisions on whether to buy the product 
+|Content|Minimal conceptual information|Detailed conceptual information on product/domain| No/less conceptual information. Instead focuses on the benefits and the customers using the product.
+|Focus|how-to| how-to and why|  sales
+
+## Why do I need a quickstart?
+
+A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. It can:
+- Make users comfortable using the application.
+- Work with any application version the user might use.
+- Lead to the willingness to handle complex workflows.
+- Help the users in exploring additional features.
+- Improve the customer experience, and help reduce costs by lowering the number of support requests.
+
+## Tips for writing a quickstart
+
+- Highlight the primary feature of your application.
+- Document the quickest/easiest way to implement the primary feature  of your application.
+- Pick a use case that the user can complete within 1-2 hours with a preference for a shorter time.
+-  Avoid complicating the quickstart by including error scenarios/complex use cases.
+- Lengthy quickstarts can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the quickstart.
+- For code samples, ensure that you include:  
+ - Any required `import` or `using` statements.
+ - Code comments that explain what the code does.
+
+## About the "Metadata" fields
+
+For more information on how to update the metadata, see [Document version](../base/base-guide.md#document-version), [Last Updated](../base/base-guide.md#last-updated), and [Application Version](../base/base-guide.md#application-versions) on the Good Docs Projects Base template.
+
+## About the "Overview" section
+
+Use this section to provide the following:
+
+- A short description of your application and its purpose.
+- A description of what the user can accomplish in this quickstart.
+- The intended audience for this document. 
+- The basic knowledge that you expect the user to have before using this quickstart.
+
+Defining the audience right in the beginning helps the users identify if the quickstart is relevant for their usage.
```

Comment:
**Suggestion (organization, non-blocking):** I agree with your tip here about the importance of defining the audience. However, this feels like a strange place to put that sentence. Earlier in this review, I made the suggestion that you should have a pre-writing or "Before you write" section. This information could be included in that section.

Reply:
Fixed